lib/quiet_quality/executors/execcer.rb

```diff
@@ -0,0 +1,44 @@
+module QuietQuality
+  module Executors
+    class Execcer
+      include Logging
+
+      def initialize(tool_options:, changed_files: nil)
+        @tool_options = tool_options
+        @changed_files = changed_files
+      end
+
+      def exec!
+        if runner.exec_command
+          Kernel.exec(*runner.exec_command)
+        else
+          info "This runner does not believe it needs to execute at all."
+          info "This typically means that it was told to target changed-files, but no relevant"
+          info "files were changed."
```

Comment:
```suggestion
          info <<~EOL
            This runner does not believe it needs to execute at all.
            This typically means that it was told to target changed-files, but no relevant
            files were changed.
          EOL
```

Reply:
I _think_ that'll end up the same? But only because I did some work splitting log messages up and prefixing stuff onto each line [here](https://github.com/nevinera/quiet_quality/blob/main/lib/quiet_quality/logger.rb#L56). I'll update.